configuration/analyzer_config.json

```diff
@@ -422,6 +422,12 @@
     },
     "soft_time_limit": 30
   },
+  "Manalyze": {
+    "type": "file",
+    "description": "A static analyzer for PE files",
+    "python_module": "manalyze.Manalyze",
+    "soft_time_limit": 500
```

Comment:
we should decrease the `max_tries` and then the `soft_time_limit` as well

Reply:
Okay, I am changing `max_tries` to 10 and `soft_time_limit` to 100 and `timeout` to 60.

Reply:
`soft_time_limit` should be made equal to about 12 minutes. Since, the timeout is 9 minutes and total polling time is 10 minutes.

Reply:
I didn't quite understand it. I should decrease the `max_tries`, Right?. Could you give exact values of `soft_time_limit`, `max_tries` and `timeout` please?

Reply:
`soft_time_limit=720`; every other value would remain same as before

Reply:
>  Okay, I am changing max_tries to 10 and soft_time_limit to 100 and timeout to 60.

ah, just saw this. No, keep the values same as before.

Reply:
Yeah, I only changed `soft_time_limit` to 720.